roles/contiv_network/templates/aci_gw.j2

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+usage="$0 start"
+if [ $# -ne 1 ]; then
+    echo USAGE: $usage
+    exit 1
+fi
+
+case $1 in
+start)
+    set -e
+
+    /usr/bin/docker run --net=host \
+    -e "APIC_URL: {{ apic_url }}" \
+    -e "APIC_USERNAME: {{ apic_username }}" \
+    -e "APIC_PASSWORD: {{ apic_password }}" \
+    -e "APIC_LEAF_NODE: {{ apic_leaf_nodes }}" \
+    --restart=unless-stopped \
```

Comment:
We need to use the `Restart` policy of systemd instead otherwise systemd will consider the service to be dead as soon as container dies and restarts as a different process

You can remove the `--restart` flag and add something on lines of the following to the unit file
```
Restart=on-failure
RestartSec=10
```